Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,70 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * The CameraLimiter is used to limit a {@link Camera}'s position (using a bounding object).
+     *
+     * If set, the camera of this limiter will be constrained by location.
+     * The camera ignores position limits in {@link SceneMode#SCENE2D} view and {@link SceneMode#COLUMBUS_VIEW}.
+     *
+     * @alias CameraLimiter
+     * @constructor
+     *
+     * @param {AxisAlignedBoundingBox|BoundingSphere|OrientedBoundingBox} [options.boundingObject] Assigned to the boundingObject attribute which can be used to specify limits on locations.
```

Comment:
Duplicate the description you use on `boundingObject` below.

Reply:
did this

Reply:
This is still not the same as this line: https://github.com/AnalyticalGraphicsInc/cesium/pull/6172/files#diff-b1419f5d132d13190f35b5c255c34a0eR34